plugins/modules/dns_view.py

```diff
@@ -0,0 +1,4348 @@
+                    key:
+                        description:
+                            - "The resource identifier."
+                        type: str
+                    name:
+                        description:
+                            - "TSIG key name, FQDN."
+                        type: str
+                    secret:
+                        description:
+                            - "TSIG key secret, base64 string."
+                        type: str
+    use_forwarders_for_subzones:
+        description:
+            - "Optional. Use default forwarders to resolve queries for subzones."
+            - "Defaults to I(true)."
+        type: bool
+    use_root_forwarders_for_local_resolution_with_b1td:
+        description:
+            - "I(use_root_forwarders_for_local_resolution_with_b1td) allows DNS recursive queries sent to root forwarders for local resolution when deployed alongside BloxOne Thread Defense. Defaults to I(false)."
+        type: bool
+    zone_authority:
+        description:
+            - "Optional. ZoneAuthority."
+        type: dict
+        suboptions:
+            default_ttl:
+                description:
+                    - "Optional. ZoneAuthority default ttl for resource records in zone (value in seconds)."
+                    - "Defaults to 28800."
+                type: int
+            expire:
+                description:
+                    - "Optional. ZoneAuthority expire time in seconds."
+                    - "Defaults to 2419200."
+                type: int
+            mname:
+                description:
+                    - "Defaults to empty."
+                type: str
+            negative_ttl:
+                description:
+                    - "Optional. ZoneAuthority negative caching (minimum) ttl in seconds."
+                    - "Defaults to 900."
+                type: int
+            refresh:
+                description:
+                    - "Optional. ZoneAuthority refresh."
+                    - "Defaults to 10800."
+                type: int
+            retry:
+                description:
+                    - "Optional. ZoneAuthority retry."
+                    - "Defaults to 3600."
+                type: int
+            rname:
+                description:
+                    - "Optional. ZoneAuthority rname."
+                    - "Defaults to empty."
+                type: str
+            use_default_mname:
+                description:
+                    - "Optional. Use default value for master name server."
+                    - "Defaults to true."
+                type: bool
+
+extends_documentation_fragment:
+    - infoblox.bloxone.common
+"""  # noqa: E501
+
+EXAMPLES = r"""
+  - name: Create a view
+    infoblox.bloxone.dns_view:
+      name: "view1"
+      state: "present"
+
+  - name: Create a view with tags
+    infoblox.bloxone.dns_view:
+      name: "view1"
+      state: "present"
+      tags:
+        location: "my-location"
```

Comment:
instead of this, can you provide a more detailed example with most of the fields populated?

Reply:
I have added a quite a few fields . Let me know if I should add more !

Reply:
for now, it is okay. We might have to redo the examples at some point maybe.